akka-discovery/src/test/scala/akka/discovery/dns/DnsDiscoverySpec.scala

```diff
@@ -48,7 +48,7 @@ class DnsDiscoverySpec extends AkkaSpec(DnsDiscoverySpec.config) with DockerBind
 
   val systemWithAsyncDnsAsResolver = ActorSystem("AsyncDnsSystem", configWithAsyncDnsResolverAsDefault)
 
-  private def testSrvRecords(discovery: ServiceDiscovery): Unit = {
+  private def testSrvRecords(discovery: ServiceDiscovery) = {
```

Comment:
or end the method with `()`? (only a test so doesn't matter)

Reply:
I've removed this option for tests now as it is very common